spec/cnb/monthly_rates_spec.rb

```diff
@@ -0,0 +1,64 @@
+require 'spec_helper'
+require 'webmock/rspec'
+require 'timecop'
+
+RSpec.describe CNB::MonthlyRates do
+  let(:monthly_rates) { CNB::MonthlyRates.new }
+  let(:currency) { 'ALL' }
+  let(:dates) do
+    {
+      invalid_past: Date.today - (365 * 11),
+      future: Date.today + 100
+    }
+  end
+
+  describe '#rate_for' do
+    before do
+      Timecop.freeze('25.02.2014')
+      url = CNB::MONTHLY_URL + Date.today.strftime('%m') +
+            CNB::MONTHLY_SEP + Date.today.strftime('%Y')
+      stub_request(:get, url)
+        .to_return(status: 200, body: load_fixture)
+    end
+
+    after do
+      Timecop.return
+    end
+
+    context 'valid parameters' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.